sublime-package.json

```diff
@@ -2,297 +2,323 @@
-									"scope": "window"
-								},
-								"xml.server.workDir": {
-									"type": "string",
-									"default": "~/.lemminx",
-									"description": "Set a custom folder path for cached XML Schemas. An absolute path is expected, although the ~ prefix (for the user home directory) is supported.",
-									"scope": "window"
-								},
-								"xml.validation.noGrammar": {
-									"type": "string",
-									"enum": [
-										"ignore",
-										"hint",
-										"info",
-										"warning",
-										"error"
-									],
-									"default": "hint",
-									"description": "The message severity when a document has no associated grammar.",
-									"scope": "window"
-								},
-								"xml.validation.enabled": {
-									"type": "boolean",
-									"default": true,
-									"description": "Enable/disable all validation.",
-									"scope": "window"
-								},
-								"xml.validation.schema": {
-									"type": "boolean",
-									"default": true,
-									"description": "Enable/disable schema based validation. Ignored if \"xml.validation.enabled\": false.",
-									"scope": "window"
-								},
-								"xml.validation.disallowDocTypeDecl": {
-									"type": "boolean",
-									"default": false,
-									"description": "Enable/disable if a fatal error is thrown if the incoming document contains a DOCTYPE declaration.",
-									"scope": "window"
-								},
-								"xml.validation.resolveExternalEntities": {
-									"type": "boolean",
-									"default": false,
-									"description": "Enable/disable resolve of external entities.",
-									"scope": "window"
-								},
-								"xml.symbols.enabled": {
-									"type": "boolean",
-									"default": true,
-									"description": "Enable/disable document symbols (Outline). No symbols are given if `\"xml.symbol.enabled\": false`.",
-									"scope": "window"
-								},
-								"xml.symbols.excluded": {
-									"type": "array",
-									"default": [],
-									"items": {
-										"type": "string"
-									},
-									"description": "Disable document symbols (Outline) for the given file name patterns. Updating file name patterns does not automatically reload the Outline view for the relevant file(s). Each file must either be reopened or changed, in order to trigger an Outline view reload.\n\nExample:\n[\n \"**/*LargeFile.xml\"\n]",
-									"scope": "window"
-								},
-								"xml.symbols.maxItemsComputed": {
-									"type": "integer",
-									"default": 5000,
-									"description": "The maximum number of outline symbols and folding regions computed (limited for performance reasons)."
+								"LSP": {
+									"properties": {
+										"LSP-lemminx": {
+											"$ref": "sublime://settings/LSP-lemminx#/definitions/PluginConfig"
+										}
+									}
 								}
 							}
 						}
 					}
 				}
-			}
+			},
```

Comment:
How about clean JSON without trailing comma?

Reply:
I'll do it but really we shouldn't care about that since schema allows trailing commas and those don't trigger validation errors with LSP-json. Having to track and remove them when moving and adding stuff is a pain.

Reply:
Anyway, it is a JSON file and we are all perfectionists. ;-) So it doesn't matter, whether ST is forgiving bad style, we just don't use it.